app/mobile_app/mobile_app_registry.py

```diff
@@ -21,8 +21,12 @@ def __init__(self, logger):
     def get_app(
         self,
         app_type: MobileAppType,
-    ) -> MobileApp | None:
-        return self._registry.get(app_type)
+    ) -> MobileApp:
+        try:
+            return self._registry[app_type]
+        except KeyError:
+            self.logger.warning('Attempted to use an app that is not initialized: %s', app_type)
```

Comment:
should this be an exception or critical, instead of warning?

Reply:
Updating.